jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/interfaces/asserts/PositionAssert.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.jdi.light.vuetify.interfaces.asserts;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.vuetify.interfaces.HasPosition;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public interface PositionAssert<A, E extends HasPosition> extends VuetifyAssert<E> {
+
+    @JDIAction("Assert that '{name}' has left position")
+    default A left() {
```

Comment:
I think, that better to name it leftPosition(), rightPosition() to give a better understanding of what we assert. And use it with has() instead of is()

Reply:
fixed